src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -84,13 +178,30 @@ public Spaceship getSpaceship() {
    */
   @Override
   public void simpleUpdate(float tpf) {
+    if (moveForward) {
+      observer.move(VRApplication.getFinalObserverRotation().getRotationColumn(2).mult(tpf * 8f));
+    }
+    if (moveBackwards) {
+      observer.move(VRApplication.getFinalObserverRotation().getRotationColumn(2).mult(-tpf * 8f));
+    }
+    if (rotateLeft) {
+      observer.rotate(0, 0.75f * tpf, 0);
+    }
+    if (rotateRight) {
+      observer.rotate(0, -0.75f * tpf, 0);
+    }
```

Comment:
Why the movement, we don't allow the captain to move arround in the cockpit right?